lib/pillar/type_convert/to_elixir.ex

```diff
@@ -159,23 +167,25 @@ defmodule Pillar.TypeConvert.ToElixir do
     end)
   end
 
-  defp convert_datetime_with_timezone(value, time_zone) do
-    [date, time] = String.split(value, " ")
-
-    case Code.ensure_loaded(DateTime) && function_exported?(DateTime, :new, 3) &&
-           DateTime.new(Date.from_iso8601!(date), Time.from_iso8601!(time), time_zone) do
-      false ->
-        {:error, "feature needs elixir v1.11 minimum"}
+  if Code.ensure_loaded?(DateTime) && function_exported?(DateTime, :new, 3) do
```

Comment:
I don't understand why it is better to put it in the function at runtime vs at compile time. 

Could you give an example where this would make any issues?

Reply:
In compile time there is important in which order libraries are compiled.
First I added special flag function

![image](https://user-images.githubusercontent.com/15660128/224985198-53da57d2-ed19-4b7d-986e-74db10c786d2.png)

After that, I added to my project "pillar", after I added "decimal"

![image](https://user-images.githubusercontent.com/15660128/224984861-f0599543-ffc5-41c8-a857-0e432228402f.png)

Some libs did recompile

```
==> tesla
Compiling 29 files (.ex)
Generated tesla app
==> talos
Generated talos app
Interactive Elixir (1.13.0) - press Ctrl+C to exit (type h() ENTER for help)
iex(1)> Pillar.decomal_is_loaded
false
```

Reply:
ahhhh I see, thanks for the explanation. I did not know about this issue!

What are your thoughts about making this a breaking change and make decimal non optional. I personally think that would make the most sense since the value of the column is decimal

Reply:
Let's make break change.

But many libs that I use have in dependency `decimal` with major version 1

What do you think about  dependency `{:decimal, ">= 1.0.0"}` ?

Reply:
Sure